theme/templates/index.liquid

```diff
@@ -1,11 +1,13 @@
-<h1 class="serif-xl">Nunu</h1>
+{% render 'home-header' %}
 
-{% for product in collections.frontpage.products %}
-  <div>
-    <a href="{{ product.url | within: collection }}">{{ product.title }}</a>
-    {{ product.price | money }}
-    <a href="{{ product.url | within: collection }}">
-      <img src="{{ product.featured_image.src | img_url: 'large' }}" alt="{{ product.featured_image.alt | escape }}">
-    </a>
-  </div>
-{% endfor %}
+{% assign show_featured_product = collections.frontpage.metafields.accentuate.show_featured_product %}
+
+{% if show_featured_product %}
+  <div class="grid grid--style-featured-product">
+    {% else %}
+    <div class="grid">
```

Comment:
Is there a nicer way to conditionally add classes with liquid? 🥴

Reply:
Yes, I believe you can use conditional logic inside your class definition, in this case you can do something like:

```
<div class="grid {% if show_featured_product %} grid--style-featured-product {% endif %}">
</div>
```

Reply:
Much better, thanks! :)